src/epoch.rs

```diff
@@ -1540,6 +1596,65 @@ fn gpst() {
     assert!((epoch.as_gpst_days() + 5.0).abs() < EPSILON);
 }
 
+#[test]
+fn unix() {
+    use core::f64::EPSILON;
+    let now = Epoch::from_gregorian_utc_hms(2022, 5, 2, 10, 39, 15);
+    assert!(
+        now.as_tai_seconds() > now.as_utc_seconds(),
+        "TAI is not ahead of UTC."
+    );
+    assert!((now.as_tai_seconds() - now.as_utc_seconds() - 37.0).abs() < EPSILON);
```

Comment:
I think this test and the one above are from the TAI/UTC test and can be safely removed from this function.

Reply:
removed in 8377a00cff4f50377f570afcc0c002e9ba52d069